src/Elmish.WPF.sln

```diff
@@ -57,6 +57,10 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FileDialogs.CmdMsg.Views",
 EndProject
 Project("{6EC3EE1D-3C4E-46DD-8F32-0CC8E7565705}") = "FileDialogs.CmdMsg", "Samples\FileDialogs.CmdMsg\FileDialogs.CmdMsg.fsproj", "{F095E7E4-28ED-4223-A92F-4E86922E34EF}"
 EndProject
+Project("{6EC3EE1D-3C4E-46DD-8F32-0CC8E7565705}") = "SubModelSeqMemory", "Samples\SubModelSeqMemory\SubModelSeqMemory.fsproj", "{97CAA5E1-B059-474B-9530-3CD7ECAC9675}"
```

Comment:
The changes to this file didn't make it into your first commit.

Can you move the changes to this file from your second commit to your first commit and then force push the resulting branch?